packages/contracts/test/helpers/gas/gas.ts

```diff
@@ -24,3 +25,39 @@ export class GasMeasurement {
     return gasCost
   }
 }
+
+interface percentDeviationRange {
+  upperPercentDeviation: number
+  lowerPercentDeviation?: number
+}
+
+export const expectApprox = (
```

Comment:
This function exists elsewhere in the codebase. Perhaps we deduplicate it by placing it in `core-utils` and import it everywhere from there

Reply:
Yeah, I brought that up in the description :)
>I verbatim copy pasted the definition of expectApprox from the integration tests' shared utils.
> I don't like this code duplication, but I also don't see a good reason to include it in the @eth-optimism/contracts package, so this seems like a good enough approach. But I'm open to alternative suggestions.

@tynes I'm closing this and replacing it with #1482, I'll make a note to do that there.